Makefile

```diff
@@ -446,6 +446,7 @@ $(INSTALLER):
 
 
 # convenience targets - so you can do `make config` instead of `make dist/config.img`, and `make installer` instead of `make dist/amd64/installer.img
+linuxkit: $(LINUXKIT)
```

Comment:
This is nice. Maybe it should be in `.PHONY`?

Reply:
done